site/plugins.json

```diff
@@ -330,7 +330,16 @@
     "name": "Is website vulnerable",
     "package": "netlify-plugin-is-website-vulnerable",
     "repo": "https://github.com/erezrokah/netlify-plugin-is-website-vulnerable",
-    "version": "1.0.10"
+    "version": "2.0.1",
+    "compatibility": [
+      {
+        "version": "2.0.1"
+      },
+      {
+        "version": "1.0.10",
+        "nodeVersion": "<12.0.0"
```

Comment:
The Node.js version range [seems slightly different](https://github.com/erezrokah/netlify-plugin-is-website-vulnerable/blob/28ead75ee9e41111b164cbaaea7a90c85580e0dc/package.json?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L51).
Should we put `<12.20.0` instead?

Reply:
Ah, good catch